D3D12HelloWorld/Graphics/Rendering.cs

```diff
@@ -16,19 +16,27 @@ public abstract class ResourceView {
     }
 
     public class ConstantBufferView : ResourceView {
-        public ConstantBufferView(GraphicsResource resource) : base(resource, CreateConstantBufferView(resource)) {
+        public ConstantBufferView(ConstantBufferView constantBufferView) : base(constantBufferView.Resource, constantBufferView.CpuDescriptorHandle) {
         }
 
-        public ConstantBufferView(ConstantBufferView constantBufferView) : base(constantBufferView.Resource, constantBufferView.CpuDescriptorHandle) {
+        public ConstantBufferView(GraphicsResource resource)
+            : this(resource, 0, resource.SizeInBytes, resource.GraphicsDevice.ShaderResourceViewAllocator) {
```

Comment:
Change here, this constructor now forwards to another constructor instead of calling the base class, and it passes the `ShaderResourceViewAllocator` just to retain original behaviour via this constructor.

However other refactoring further down allows for new use cases...